frontend/src/lib/components/details/top_card.svelte

```diff
@@ -70,18 +74,34 @@
       <div class="flex justify-between">
         <h2 class="card-title">{title}</h2>
         {#if imdbId}
-          <a
-            href="https://www.imdb.com/{imdbId.startsWith('tt')
-              ? 'title'
-              : 'name'}/{imdbId}"
-          >
-            <div
-              class="badge badge-secondary mx-2 transform
+          {#if imdbId.startsWith("tt")}
```

Comment:
Are there cases of it not starting with tt?

Reply:
Would you mind adding a comment? I will 100% forget that :D

Reply:
Done